SwiftUI/Model/ZimMigration.swift

```diff
@@ -34,15 +34,6 @@ enum ZimMigration {
         }
     }
 
```

Comment:
This URL mapping to custom apps is no longer needed, as this was applied "after" the interactionState was restored on the WebView, and the URL was re-set with the result of this.